electra.xcodeproj/project.pbxproj

```diff
@@ -349,7 +348,7 @@
 			isa = PBXProject;
 			attributes = {
 				LastUpgradeCheck = 0910;
-				ORGANIZATIONNAME = "Ian Beer";
+				ORGANIZATIONNAME = "Electra Contributors";
```

Comment:
I'd really prefer ElectraOrg or ElectraTeam to "Electra Contributors" :P

Reply:
but it's all up to @coolstar, as well as #25 (well that one is up to @coolstar and up to commiters to agree with chosen license)

Reply:
it can also be moved to organization account with same name

Reply:
I just didn’t like having Ian Beer everywhere

Reply:
Ikr, it was me who suggested it on discord kek

Reply:
I guess we can use contributors for now, because I’m not at my Mac to go through and find and replace. But yeah, I agree; we could make a git org for this, and submodule jailbreakd and other binaries? Just a idea